main/CMakeLists.txt

```diff
@@ -430,12 +408,9 @@ else (MINGW)
          list(APPEND dlls_to_copy "${QT_INSTALL_BINS}/Qt5WebEngineWidgets.dll" "${QT_INSTALL_BINS}/Qt5WebEngineCore.dll")
       endif(USE_WEBENGINE)
 
-      if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.14.0")
-         list(APPEND dlls_to_copy "${QT_INSTALL_BINS}/Qt5QmlModels.dll")
-      endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.14.0")
-      if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.15.0")
-         list(APPEND dlls_to_copy "${QT_INSTALL_BINS}/Qt5QmlWorkerScript.dll")
-      endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.15.0")
+      list(APPEND dlls_to_copy
+          "${QT_INSTALL_BINS}/Qt5QmlModels.dll"
+          "${QT_INSTALL_BINS}/Qt5QmlWorkerScript.dll")
```

Comment:
Now that I look one more time at it: you can actually merge this `list(APPEND dlls_to_copy` statement into the one of line 395. Formerly, the dlls were appended to the list dependently from the Qt version, but now that there is no more dependency and they are appended anyway, there is no need any more to have a separate `list(APPEND dlls_to_copy` statement for this.

Reply:
True,

Reply:
darn, now I broke it

Reply:
Ah, copy paste error...